c-data-channels/data-channels.c

```diff
@@ -0,0 +1,33 @@
+#include "webrtc.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+void on_open(GoDataChannel d)
+{
+    while (1)
+    {
+        GoSendText(d, "hello");
+        sleep(5);
+    }
+}
+
+void on_message(GoDataChannel d, struct GoDataChannelMessage msg)
+{
+    printf("%s\n", (char *)msg.data);
```

Comment:
I would keep the same format as the other ones `fmt.Printf("Message from DataChannel '%s': '%s'\n", d.Label(), string(msg.Data))`